### PR TITLE
MSI Installer Fixes

### PR DESCRIPTION
**Pull request by shivamkm07**
# Description

Fixes several issues reported in MSI Installer for CLI

## Issue reference

Please reference the issue this PR will close: #1079 , #1077 , #1076 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation

### CODE REVIEW

File: .github/wix/dapr.wxs [hunk 1]
```diff
@@ -20,22 +20,38 @@
...
 
         <!-- Dapr EXE Component -->
         <Component Id='DaprEXE_Comp' Guid='83363a24-2323-489c-b9b7-a3fd7f0dacd1'>
           <File Id='DaprEXE' Name='dapr.exe' DiskId='1' Source='$(var.DaprEXEDirectory)/dapr.exe' KeyPath='yes'>
           </File>
         </Component>
 
+        <!-- Update PATH Environment Variable -->
+        <Component Id='UpdatePATH_Comp' Guid='0198647e-bae8-47e6-b9ff-01881ee540ae'>
+        <CreateFolder />
```
Comment (shubham1172) [inline .github/wix/dapr.wxs:34]: Is `CreateFolder` needed here?

Comment (shubham1172) [reply .github/wix/dapr.wxs:34]: It's apparently required for this block to be executed.

Comment (shivamkm07) [reply .github/wix/dapr.wxs:34]: Yes

**Comment by mukundansundar** [comment]
Has this been manually validated ?

**Review by mukundansundar — APPROVED** [review]

**Comment by shivamkm07** [comment]
> Has this been manually validated ?

Yes

**Comment by shivamkm07** [comment]
> Has this been manually validated ?

For others wanting to manually validate, it can be downloaded from here: https://github.com/dapr/cli/actions/runs/3189162837 as part of published artifacts.

**Review by shubham1172 — APPROVED** [review]
LGTM, manually validated as well!

**Pull request merged**